kuma/javascript/ssr-server.js

```diff
@@ -81,7 +88,13 @@ app.get('/readiness/?', (req, res) => {
  * not be parsed, escaped, or displayed as HTML.
  */
 app.post('/ssr/:componentName', (req, res) => {
-    res.json(ssr(req.params.componentName, req.body));
+    try {
+        res.json(ssr(req.params.componentName, req.body));
+    } catch (err) {
+        console.error(`Error running ssr(): ${err.toString()}`);
+        Sentry.captureException(err);
```

Comment:
The core of this whole PR is that I have to manually trigger this or else nothing's sent to Sentry. 
I don't really understand why the `@sentry/node` and `express` configuration can't do this automatically.